Content.Server/Clothing/Components/ClothingSolutionTransferComponent.cs

```diff
@@ -0,0 +1,35 @@
+using Content.Shared.FixedPoint;
+using Robust.Shared.Localization;
+using Robust.Shared.ViewVariables;
+using Content.Shared.Actions;
+using Content.Shared.Actions.ActionTypes;
+using Content.Shared.Toggleable;
+
+namespace Content.Server.Clothing.Components
+{
+    [RegisterComponent]
+	public sealed class ClothingSolutionTransferComponent : Component
```

Comment:
All of the variables are just duplicates of other component stuff. Really all this component and entitysystem need to do is add an action on equip, otherwise it's just duplicated code from solutions and drinks